pyiron/atomistics/job/interactive.py

```diff
@@ -270,65 +268,26 @@ def interactive_time_getter(self):
     def interactive_volume_getter(self):
         return self.initial_structure.get_volume()
 
-    def get_structure(self, iteration_step=-1, wrap_atoms=True):
```

Comment:
I don't have any objection to this, but I also don't understand what the ramifications are. So don't feel any need to change anything on my account, but know that I am not wisely and with great understanding approving this change 😉

Reply:
Yeah I also don't really know what could happen, but what's already happening is bad enough (i.e. depending on the `run_mode`, `get_structure` was returning different things). Now there's only one `get_structure` in `atomistic.py`, which I hope will simply it.